addons/sale_timesheet/report/timesheets_analysis_report.py

```diff
@@ -36,6 +36,11 @@ def _select(self):
             A.so_line AS so_line,
             A.timesheet_invoice_type AS timesheet_invoice_type,
             A.timesheet_invoice_id AS timesheet_invoice_id,
-            (-1 * A.unit_amount * A.amount) AS timesheet_revenues,
-            CASE WHEN order_id IS NULL THEN 0 ELSE unit_amount END AS billable_time
+            CASE WHEN A.order_id IS NULL THEN 0 ELSE unit_amount * SOL.price_unit END AS timesheet_revenues,
+            CASE WHEN A.order_id IS NULL THEN 0 ELSE unit_amount END AS billable_time
```

Comment:
Thanks 🙂 
I would make it more explicit.
```suggestion
            CASE WHEN A.order_id IS NULL THEN 0 ELSE A.unit_amount * SOL.price_unit END AS timesheet_revenues,
            CASE WHEN A.order_id IS NULL THEN 0 ELSE A.unit_amount END AS billable_time
```